packages/kbn-plugin-generator/sao_template/template/server/routes/index.ts

```diff
@@ -0,0 +1,17 @@
+import { IRouter } from '<%= relRoot %>/../src/core/server';
+
+export function defineRoutes(router: IRouter) {
+  router.get(
+    {
+      path: '/api/<%= snakeCase(name) %>/example',
+      validate: false,
+    },
```

Comment:
Should we add an example usage of validation, to demonstrate how the `@kbn/config-schema` validation works?

Reply:
I think we should keep the generator basic.
We will refer users to a library of sample plugins for more examples.